src/pages/interactive.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
```

Comment:
We probably shouldn't be exposing this page, it's not a big deal since there's nothing linking to it, but I think it's a good idea to keep the pages clean and to a required minimum

Reply:
ah yeah, I meant to remove that